eacl/target.go

```diff
@@ -37,23 +37,25 @@ func (t Target) CopyTo(dst *Target) {
 	}
 }
 
-// BinaryKeys returns list of public keys to identify
+// BinaryKeys returns list of public keys or accounts to identify
 // target subject in a binary format.
 //
-// Each element of the resulting slice is a serialized compressed public key. See [elliptic.MarshalCompressed].
-// Use [neofsecdsa.PublicKey.Decode] to decode it into a type-specific structure.
+// Each element of the resulting slice is a serialized compressed public key or account from the public key.
+// See [elliptic.MarshalCompressed], [keys.PublicKey.GetScriptHash].
+// Use [neofsecdsa.PublicKey.Decode] to decode it into a type-specific structure of len equals 33 or [address.StringToUint160]
+// in other case.
 //
 // The value returned shares memory with the structure itself, so changing it can lead to data corruption.
 // Make a copy if you need to change it.
 func (t *Target) BinaryKeys() [][]byte {
 	return t.keys
 }
 
-// SetBinaryKeys sets list of binary public keys to identify
+// SetBinaryKeys sets list of binary public keys or accounts to identify
```

Comment:
I've added SetAccounts/Accounts methods. But I think we don't need to deprecate SetKeys methods because they are still valid and required. How I understand the idea, they still required.
Moreover, I am considering adding length checks to be sure it is an real account data. What do you think? @cthulhu-rider

Reply:
we provide `SetAccount` / `Accounts` and write in docs that argument _can be_ binary-encoded public key. With this, `*BinaryKeys` become obsolete so we deprecated them

see also https://github.com/nspcc-dev/neofs-sdk-go/pull/592#discussion_r1666734549